assets/src/edit-story/components/form/test/swtich.js

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
You want to test the callback too, besides the checked attributes.

```js
const onChange = jest.fn();
const { getByText, getByLabelText } = render(<Switch onChange={onChange} />, {
  wrapper: ThemeProviderWrapper,
});

fireEvent.click(getByText(onLabel));
expect(onChange).toHaveBeenCalledWith(true);
```

Also add a test where you initially set it to true and check that it correctly displays as true. Then update to false (via `rerender`) and check it correctly changed internal state.

Futrthermore, the filename is misspelled: `swtich` rather than `switch`.

Reply:
Added.